packages/react/src/hooks/useDidUpdate.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react';
+
+/**
+ *  useDidUpdate hook
+ *
+ *  dependency 배열의 요소가 업데이트 됐을때 콜백 함수를 실해압니다.
+ *
+ * @param {Function} callback dependency 배열의 요소가 업데이트 될 때 호출 할 콜백 함수
+ * @param {Array} conditions 업데이트를 트리거하는 변수 목록
+ */
+export default function useDidUpdate(callback: () => any, conditions?: any[]) {
```

Comment:
```suggestion
export default function useDidUpdate(callback: () => any, dependencies?: DependencyList) {
```

요기는 dependencies가 맞는 것 같습니다!

Reply:
`DependencyList` 메모.... 변경했습니당
https://github.com/Lubycon/lubycon-frontend-libraries/pull/41/commits/ba2fb04aa6b33c2ba8ede17ffd005c53f494eec5